hwdb.d/70-pointingstick.hwdb

```diff
@@ -96,22 +96,27 @@ evdev:name:*[tT]rack[pP]oint*:dmi:bvn*
 
 # Latitude D620
 evdev:name:*DualPoint Stick:dmi:bvn*:bvr*:bd*:svnDellInc.:pnLatitudeD620*:*
+ ID_INPUT_POINTINGSTICK=1
```

Comment:
Hmm, but is this a good change? The docs say "In case multiple records match a given lookup string, the key-value pairs from all records are combined.". So unless that stopped being true, or there is some additional consideration, I think this change should be dropped.